example/src/functions.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * @module
```

Comment:
Thanks for the explanation. It's a bit unintuitive that the comment still wouldn't have appeared even if it was formatted correctly. Are you saying that if  `index.ts` is your only entry point,  `@module` only works in `index.ts`?

Anyway, I decided to remove the `@module` tag since I didn't like how it looked on the main index page.

Reply:
Not necessarily - if you exported with `export * as Functions from "./functions"` in the entry point, then there would be a namespace called `Functions`, which would have this comment attached to it.